web/src/reducers/dataReducer.js

```diff
@@ -132,32 +132,14 @@ const reducer = (state = initialDataState, action) => {
         // By default hasData is only true if there is a parser - here we overwrite that value
         // if there is data despite no parser (for CONSTRUCT_BREAKDOWN estimation models)
         zone.hasData = zone.hasParser || !hasNoData;
-
-        // Validate data
-        modeOrder.forEach((mode) => {
-          if (mode === 'other' || mode === 'unknown' || !zone.datetime) { return; }
-          // Check missing values
-          // if (country.production[mode] === undefined && country.storage[mode] === undefined)
-          //    console.warn(`${key} is missing production or storage of ' + mode`);
-          // Check validity of production
-          if (zone.production[mode] !== undefined && zone.production[mode] < 0) {
-            console.warn(`${key} has negative production of ${mode}`);
-          }
-          // Check load factors > 1
-          if (zone.production[mode] !== undefined
-            && (zone.capacity || {})[mode] !== undefined
-            && zone.production[mode] > zone.capacity[mode]) {
-            console.warn(`${key} produces more than its capacity of ${mode}`);
-          }
-        });
```

Comment:
This validation block is either redundant with what we already do in the backend, or simply not useful. It produces a lot of logs, and I recommend removing them to reduce the clutter.

Reply:
One could argue that it might be helpful for users/contributors to identify zones that needs updated capacities... But not sure if anyone ever uses it, so I'm fine with removing it :)

Reply:
good point.